docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md

```diff
@@ -178,6 +183,64 @@ To configure federated authentication in YugabyteDB Aeon, do the following:
 1. Enter the Okta domain for your application.
 1. Click **Enable**.
 
+You are redirected to sign in to your IdP to test the connection. After the test connection is successful, federated authentication is enabled.
+
+  {{% /tab %}}
+
+  {{% tab header="JumpCloud" lang="jumpcloud" %}}
+
+**Create an application in JumpCloud**
+
+To use JumpCloud for your IdP, do the following:
+
+1. Sign in to your JumpCloud account and create an application.
+
+    - Sign in using an administrator account.
+    - Under **SSO Applications**, click **Add New Application**.
+    - Select **Custom Application**, and make sure the integration supports "SSO with OIDC" on the next page.
+    - Under **Manage Single Sign-On (SSO)**, select **Configure SSO with OIDC**, and click **Next**.
+    - Under **Enter General Info**, add the application name (for **Display Label**), **Description**, and logo (for **User Portal Image**), and select **Show this application in User Portal**.
+
+    This information is displayed as a tile when users sign in to YugabyteDB Aeon.
+    - Click **Save Application**.
+
+1. Configure your application.
+
+    Under **Endpoint Configuration**, do the following:
+
+    - **Redirect URIs** - enter `https://yugabyte-cloud.okta.com/oauth2/v1/authorize/callback`.
+    - **Client Authentication Type** - select **Client Secret Post**.
+    - **Login URL** - enter `https://cloud.yugabyte.com/login`.
+
+    Under **Attribute Mapping**, for **Standard Scopes**, select **Email** and **Profile**.
+
+    Click **Activate** when you are done.
+
+    You will be prompted in a pop up to save the **Client ID** and **Client Secret**. Save these in a secure location, you will need to provide these credentials in YugabyteDB Aeon.
+
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in JumpCloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Aeon, and click **Save** when you are done.
+
+To configure JumpCloud federated authentication in YugabyteDB Managed, you need the following application properties:
+
+- Client ID and secret of the application you created. Note that you will get a pop up to save the **Client ID** and **Client Secret** when you activate your application. The **Client ID** is also displayed on the **SSO** tab.
+- The Authorization URL for your application. [where is this displayed?]
```

Comment:
Need to add where they can find this - where does the Authorization URL come from/where do they find it?
Is it the Redirect URI they entered above, or something else?

Reply:
@Aastha-yb

Reply:
Authorization URL is not required. We should remove this point.

Reply:
No Authorization URL needed. This was initially discussed to be added and exists in the POC doc, but later we decided to not keep it as it is a constant URL. Pls refer the following ss for more context on the info needed at YBM side : 
![Uploading Screenshot 2024-06-25 at 11.57.26 AM.png…]()